src/graph/backend/dnnl/op_executable.cpp

```diff
@@ -1086,7 +1086,7 @@ concat_executable_t::desc_t concat_executable_t::create_desc(
     const auto rank = op->get_output_value(0)->get_logical_tensor().ndims;
     const auto res = utils::try_reverse_axis(
             op->get_attr<int64_t>(op_attr::axis), rank);
-    assertm(res.first, "Incorrect axis value.");
+    if (!res.first) { BACKEND_DNNL_ENFORCE(0, "Unsupported resampling mode."); }
```

Comment:
why not verbose check?

Reply:
If I understand it correctly, `BACKEND_DNNL_ENFORCE` is essentially an assertion and only takes effect in debug build. Is it the purpose here?

Reply:
You're right. We need some hints even in release mode. I added a check macro with no return value. Could you review it again?